notebooks/implicit_differentiation.ipynb

```diff
@@ -0,0 +1,561 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "protecting-marine",
+   "metadata": {},
+   "source": [
+    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/google/jax-md/blob/main/notebooks/implicit_differentiation.ipynb)"
+   ]
+  },
+  {
+   "cell_type": "code",
```

Comment:
Can you hide the code contents of this cell by default (by clicking on the three vertical dots on the right hand side of the cell when it's selected and then clicking Form->Hide Code)?

Reply:
Done